src/styles/index.scss

```diff
@@ -2,2 +1,90 @@
+* {
+  padding: 0;
   margin: 0;
+  box-sizing: border-box;
+}
+
+$size: 100px;
+$border: 1px solid black;
+$background: #eee;
+$text-color: #000;
+$text-size: 30px;
+$gap: 1px;
+$hover-color: #ffbfcb;
+$transition-duration: 0.5s;
```

Comment:
It's good practice to group your variables into a separate partial file (e.g., _variables.scss) and import it. This helps to keep the code organized and makes it easier to manage your variables.